### PR TITLE
test technique la fourchette

### PR DESCRIPTION
**Pull request by suarezd**
Thanks for the test it was interesting and i hope i have successfully solved the issue. :)

### CODE REVIEW

**Comment by dav-m85** [comment]
@suarezd Nice PR ! We will review it on Monday. Have a Nice weekend :)

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="./config/bootstrap_test.php" colors="true">
+<phpunit bootstrap="./tests/bootstrap_test.php" colors="true">
```
Comment (dav-m85) [inline phpunit.xml.dist]: :+1:

File: src/LaFourchette/Logger/DummyLogger.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?php
+/**
```
Comment (dav-m85) [inline src/LaFourchette/Logger/DummyLogger.php]: Can you remove these auto-added headers from your files? Git already track authors right?

File: src/LaFourchette/Logger/DummyLogger.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ * Date: 11/04/2015
+ * Time: 10:30
+ */
+
+namespace LaFourchette\Logger;
+
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+
+class DummyLogger
```
Comment (dav-m85) [inline src/LaFourchette/Logger/DummyLogger.php]: Please add a phpdoc describing the role fo your class. DummyLogger is not explicit enough.

File: src/LaFourchette/Logger/DummyLogger.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+     */
+    public static function getLogFile($logDir = null)
+    {
+        if (!$logDir) {
+            $logDir = __DIR__ . '/../../../logs';
+        }
+
+        return $logDir . '/' . self::LOG_FILE_MASK . '.log';
+    }
+}
```
Comment (dav-m85) [inline src/LaFourchette/Logger/DummyLogger.php]: Trailing EOL. A good habit is to run ```php-cs-fixer fix ...``` to correct PSR-2 code style. cf https://github.com/FriendsOfPHP/PHP-CS-Fixer

**Comment by suarezd** [comment]
Thank you again @dav-m85, i'm fixing it then i'll submit a new PR if i may so :)

**Comment by dav-m85** [comment]
@suarezd Just add commits to your master, this PR will automagically updates (github's power).

**Comment by suarezd** [comment]
It's done @dav-m85 , thanks for the help ! :)

**Comment by dav-m85** [comment]
Related to https://github.com/lafourchette/prototype/issues/5

**Comment by suarezd** [comment]
Hi @dav-m85 , i hope you're doing well.
Does  #5 mean that i've the opportunity to continue the test ? :)

**Comment by dav-m85** [comment]
That's not necessary. I rephrased the issue that's all.

**Comment by dav-m85** [comment]
@suarezd, can you please merge back master ? I added travis and scrutinizer for code analysis :)

**Comment by suarezd** [comment]
Hi @dav-m85, job's done ;) thoses tools seems nice !

**Pull request closed without merging**